Source/Core/VideoBackends/Software/EfbInterface.cpp

```diff
@@ -538,6 +529,10 @@ void CopyToXFB(yuv422_packed* xfb_in_ram, u32 fbWidth, u32 fbHeight, const EFBRe
       scanline[i] = GetColorYUV(x, y);
     }
 
+    // Flipper clamps the border colors
```

Comment:
Now that I think about it, I really should have put this change in it's own PR.

My bad.